doc/troubleshooting.md

```diff
@@ -0,0 +1,362 @@
+# Troubleshooting compliance-operator
+This document describes how to troubleshoot problems with compliance
+operator. The information can be useful either to diagnose a problem or
+provide information in a bug report.
+
+Please refer to the README for general information about the API objects.
+
+## General tips
+
+   * The Compliance Operator emits Kubernetes events when something
+     important happens. You can either view all events in the cluster (`oc get events
+     -nopenshift-compliance`) or events for an object, e.g. for a scan
+     (`oc get compliancescan/$scan`)
```

Comment:
wouldn't you want to use `oc describe <object>/<name>`? since that outputs the events as well for that object/

Reply:
of course, fixed